language/google/cloud/language/client.py

```diff
@@ -45,10 +46,16 @@ class Client(client_module.Client):
     SCOPE = ('https://www.googleapis.com/auth/cloud-platform',)
     """The scopes required for authenticating as an API consumer."""
 
-    def __init__(self, credentials=None, _http=None):
+    _CONNECTION_CLASSES = {
+        'v1': Connection,
+        'v1beta2': V1Beta2Connection,
+    }
+
+    def __init__(self, credentials=None, api_version='v1', _http=None):
         super(Client, self).__init__(
             credentials=credentials, _http=_http)
-        self._connection = Connection(self)
+        ConnectionClass = self._CONNECTION_CLASSES[api_version]
```

Comment:
1. Are we worried about an unwrapped `KeyError` (if the wrong `api_version` is used)?
1. Human lint: shouldn't it be `connection_class`?